test/integration/options.spec.js

```diff
@@ -91,6 +92,29 @@ describe('options', function () {
     });
   });
 
+  describe.only('--file', function () {
+    before(function () {
```

Comment:
a good rule of thumb is that you should always use `beforeEach` unless you can't.